### DISCUSSION TITLE
表格组件的工具条新建和编辑窗口弹出的位置不居中,有属性控制吗？

### DISCUSSION THREAD

**Discussion by yangf85** [dotnetcore/BootstrapBlazor#3146] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

当设置了EditDialogIsDraggable的时候，就不居中了，不设置是居中的

**Comment by bb-auto** [comment]
@yangf85 Thank you for reporting. We will give triage later.

**Comment by ArgoZhang** [comment]
@yangf85 请提供截图 画框 箭头 指示哪里居中？ 帮忙 star 一下我们的仓库

**Comment by yangf85** [comment]
> @yangf85 请提供截图 画框 箭头 指示哪里居中？ 帮忙 star 一下我们的仓库

![image](https://github.com/dotnetcore/BootstrapBlazor/assets/42733580/ba1f713f-bfad-482a-a830-db08da11f5f5)
这是没有设置可以拖动的时候，弹窗是居中的
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/42733580/c54b52b1-18bf-43fa-bd7a-3798bbfdc419)
这是可以拖动的时候，弹窗跑上面去了，
**已经Star了**

- 另外为什么使用DataService 的时候，不做任何操作，点击弹窗关闭，也会添加一个空对象，而且表格也不会刷新这个添加的这个对象，怎么处理？ 
- 
- 还有表单的验证逻辑是在EditFooterTemplate处理？

**Comment by ArgoZhang** [comment]
这是正常的。你看看样式就知道了。拖动的时候没有设置 垂直居中的样式

**Reply by yangf85** [reply]
我的意思弹出的时候，应该让这个对话框居中，现在弹出的时候就跑上面了

**Reply by ArgoZhang** [reply, marked as answer]
我理解你的意思，这两个样式是冲突的，没有解决，这是 bs 的问题。我觉得不是大问题，没花精力解决
